docs/developer/environment.rst

```diff
@@ -14,7 +14,18 @@ To setup a local development environment, the developers should follow the steps
 * Install ``anaconda`` (``miniconda`` is recommended)
 * Clone the repository and make a feature branch based off ``next``.
 * Create a new virtual environment with ``conda env create`` which uses ``environment.yml``
+
+.. code-block:: sh
+
+   conda env create --file environment.yml
```

Comment:
By default, `conda env create` will read `environment.yml` already. Being explicit isn't a bad thing though.

Reply:
so `conda env create` looks for a file with the name "environment.yml" or _any_ YML file that looks like a conda environment?